css-shared-element-transitions/index.bs

```diff
@@ -948,4 +942,9 @@ A <dfn>captured element</dfn> is a [=struct=] with the following:
             1. At the [=user-agent origin=],
                 set |incoming|'s 'object-view-box' property to a value that when applied to |incoming|,
                 will cause the view box to coincide with [=incoming element=]'s [=border box=] in the image.
+    
+            1. The [=incoming element=] and its contents (the flat tree descendants of the element,
+                including both text and elements, or the replaced content of a replaced element) are not painted
+                (as if they had visibility: hidden) and do not respond to hit-testing (as if they had pointer-events: none)
+                until |incoming| exists.
```

Comment:
What does it mean until |incoming| exists? Don't they not paint for the duration of the transition since the paint is applied to the pseudo element instead?

Reply:
"Don't they not paint for the duration of the transition since the paint is applied to the pseudo element instead". Yes. I am adding it here because the ::page-transition-incoming-image pseudo-element exists for the same duration when the incoming element's painted output is being displayed via the pseudo element.

Reply:
We need to think through how this is defined for the root element. Technically ::page-transition and the pseudo sub-tree are flat tree descendants of the element of the root element. But if the root is participating in the transition then we shouldn't be hit-testing other descendants of the root element.